src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,106 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
+import model.Person.Sex;
 
 public class StreamPractice {
+
     /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
+     * Given list of strings where each element contains 1+ numbers: input = {"5,30,100", "0,22,7",
+     * ...} return min integer value. One more thing - we're interested in even numbers. If there is
+     * no needed data throw RuntimeException with message "Can't get min value from list: < Here is
+     * our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+            .flatMap(s -> Arrays.stream(s.split(",")))
+            .mapToInt(Integer::valueOf)
+            .filter(i -> i % 2 == 0)
+            .min().orElseThrow(() ->
```

Comment:
```suggestion
            .min()
            .orElseThrow(() ->
```
put each on its own line

Reply:
Changes have been made